ndk/src/surface_texture.rs

```diff
@@ -32,6 +32,11 @@ impl SurfaceTexture {
     ///
     /// This function should be called with a healthy JVM pointer and with a non-null surface texture,
     /// which must be kept alive on the Java/Kotlin side.
+    /// Get a reference to the native ASurfaceTexture from the corresponding java object.
```

Comment:
This doesn't belong under `# Safety`, and a newline is otherwise missing that would fold this sentence into the previous pagraph.